test-integration/test_integration/conftest.py

```diff
@@ -72,8 +77,14 @@ def setup(self):
     @cog.input("path", type=Path)
     @cog.input("output_file", type=bool, default=False)
     def predict(self, text, path, output_file):
+        logging.warn("writing log message")
+        time.sleep(.1)
```

Comment:
I know this only amounts to an extra 0.4 seconds in test duration and doesn't really matter, but is there another way to do this that would run faster? Does python have something like Node's `process.nextTick()`?

Reply:
I'm not sure how to make that work since there are two processes reading from stderr and stdout. Without explicit synchronization it would be hard to have them arrive in order without a small timeout.